### PR TITLE
Add WordPress.Security.SafeRedirect XML documentation

### PR DESCRIPTION
**Pull request by marconmartins**

### CODE REVIEW

File: WordPress/Docs/Security/SafeRedirectStandard.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<documentation title="Safe Redirect">
+    <standard>
+    <![CDATA[
+    Safe redirects must be used.
```
Comment (GaryJones) [inline WordPress/Docs/Security/SafeRedirectStandard.xml]: I'd like to see this expanded - _why_ must safe redirects be used? What is a _safe_ redirect? Why does a non-safe redirect even exist? Where can I find out more about this concept? etc.

File: WordPress/Docs/Security/SafeRedirectStandard.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Safe redirect.">
+        <![CDATA[
+wp_safe_redirect( $location );
```
Comment (jrfnl) [inline WordPress/Docs/Security/SafeRedirectStandard.xml]:
```suggestion
<em>wp_safe_redirect</em>( $location );
```

File: WordPress/Docs/Security/SafeRedirectStandard.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+]]>
+        </code>
+        <code title="Invalid: Unsafe redirect.">
+        <![CDATA[
+wp_redirect( $location );
```
Comment (jrfnl) [inline WordPress/Docs/Security/SafeRedirectStandard.xml]:
```suggestion
<em>wp_redirect</em>( $location );
```

**Comment by jrfnl** [comment]
@marconmartins Just wondering if you'll have a chance to finish this off in the near future.
I'd like to release WPCS 2.2.0 soon and it would be great if this PR could be included.

If you haven't got time or lost interest, please let us know and we'll see if we can find someone to take over.

**Pull request closed without merging**